### PR TITLE
Rytis/readme update

### PR DESCRIPTION
**Pull request by RytisSapka**
Not quite sure how the readme file should look so any guidelines would be great.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +12,30 @@ Contains the __@itwin/insights-client__ package that wraps sending requests to t
```
Comment (MindaugasButkus) [inline README.md:12]: Update the `About this Repository` section. This package sends requests to the Reporting service, not to the project service.

File: README.md [hunk 2]
```diff
@@ -12,17 +12,30 @@ Contains the __@itwin/insights-client__ package that wraps sending requests to t
 
 ## Environment Variables
 
-```
-# ---- Optional URL prefix for dev/qa environments ----
+### Optional URL prefix for dev/qa environments
```
Comment (MindaugasButkus) [inline README.md:15]: Keep all the `.env` file entries in a code block, so it is easier to copy.

File: README.md (hunk 1 above)
Comment (RytisSapka) [reply README.md:12]: changed project to reporting

File: README.md (hunk 2 above)
Comment (RytisSapka) [reply README.md:15]: made a code block

File: README.md [hunk 3]
```diff
@@ -8,21 +8,36 @@ If you have questions, or wish to contribute to iTwin.js, see our [Contributing
...
 IMJS_URL_PREFIX=""
 
-# ----Authorization for running tests ----
-IMJS_OIDC_BROWSER_TEST_CLIENT_ID=""
-IMJS_OIDC_BROWSER_TEST_REDIRECT_URI=""
-IMJS_OIDC_BROWSER_TEST_SCOPES=""
-IMJS_TEST_REGULAR_USER_NAME=""
-IMJS_TEST_REGULAR_USER_PASSWORD=""
+### Authorization for running tests
+- Create .env file and configure the following variables.
```
Comment (MindaugasButkus) [inline README.md:19]:
```suggestion
- Create .env file and configure the following variables:
```

File: README.md [hunk 4]
```diff
@@ -8,21 +8,36 @@ If you have questions, or wish to contribute to iTwin.js, see our [Contributing
...
 
-Contains the __@itwin/insights-client__ package that wraps sending requests to the project service. Visit the [Insights API](https://developer.bentley.com/apis/insights/) and [Carbon calculation API](https://developer.bentley.com/apis/carbon-calculation/) for more documentation on the insights service.
+Contains the __@itwin/insights-client__ package that wraps sending requests to the reporting service. Visit the [Insights API](https://developer.bentley.com/apis/insights/) and [Carbon calculation API](https://developer.bentley.com/apis/carbon-calculation/) for more documentation on the insights service.
 
 ## Environment Variables
 
-```
-# ---- Optional URL prefix for dev/qa environments ----
+### Optional URL prefix for dev/qa environments
 IMJS_URL_PREFIX=""
```
Comment (MindaugasButkus) [inline README.md:16]: Surround IMJS_URL_PREFIX in triple ` quotes

File: README.md (hunk 3 above)
Comment (RytisSapka) [reply README.md]: changed to ```:```

File: README.md (hunk 4 above)
Comment (RytisSapka) [reply README.md:17]: surrounded (in the line above and below)

**Comment by MindaugasButkus** [comment]
Bump the patch version in package.json so that the package with changed README gets deployed.

**Review by MindaugasButkus — APPROVED** [review]

**Pull request merged**